### PR TITLE
Move core CRDs to cluster-essentials release

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- Move core CRDs to cluster-essentials release

**Related issue(s)**

See also: #7933

### CODE REVIEW

File: resources/cluster-essentials/files/crd-kubeless.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (michal-hudy) [inline resources/cluster-essentials/files/crd-kubeless.yaml:2]: This CRD no longer exists in Kyma, it has been removed.

Comment (m00g3n) [reply resources/cluster-essentials/files/crd-kubeless.yaml:2]: thx I will update the PR

**Pull request merged**